### PR TITLE
Update manifest-sources-for-serverless-lambda.md

### PR DESCRIPTION
**Pull request by roshnisarangadharan**

### CODE REVIEW

File: docs/continuous-delivery/deploy-srv-diff-platforms/serverless/manifest-sources-for-serverless-lambda.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ Below describes setting configuration parameters for AWS S3 bucket as a manifest
 4. **File/Folder Path**: Enter the path from the root of the bucket to the `serverless.yml` file.
 
 :::info
-The file path here can be a `.zip` file. Harness will automatically decompress the file when it is pulled from the bucket. 
+Harness supports `.zip` files only. For users using Serverless V1, Harness will automatically decompress the file and save it to a folder when it is pulled from the bucket. Users using the containerized version of Serverless, i.e., Serverless V2, must decompress the `serverless.yml` and other files at the root level.
```
Comment (piyush-bhuwalka) [inline docs/continuous-delivery/deploy-srv-diff-platforms/serverless/manifest-sources-for-serverless-lambda.md:48]: Harness expects the serverless manifest files to be zipped inside the folder for Serverless V1

For Serverless V2, serverless manifest files should be at the root and not zipped inside a folder

Comment (roshnisarangadharan) [inline docs/continuous-delivery/deploy-srv-diff-platforms/serverless/manifest-sources-for-serverless-lambda.md:48]:
```suggestion
Harness supports `.zip` files only. Harness expects that you keep the Serverless manifest files to be zipped inside a folder for Serverless V1, and keep the files at root for Serverless V2. 
```

**Review by PiyushBhuwalka-sopho — APPROVED** [review]

File: docs/continuous-delivery/deploy-srv-diff-platforms/serverless/manifest-sources-for-serverless-lambda.md [hunk 2]
```diff
@@ -45,7 +45,7 @@ Below describes setting configuration parameters for AWS S3 bucket as a manifest
 4. **File/Folder Path**: Enter the path from the root of the bucket to the `serverless.yml` file.
 
 :::info
-The file path here can be a `.zip` file. Harness will automatically decompress the file when it is pulled from the bucket. 
+Harness supports `.zip` files only. Harness expects that you keep the Serverless manifest files zipped inside a folder for Serverless V1, and at root for Serverless V2. 
 :::
```
Comment (SmritiSatya) [inline docs/continuous-delivery/deploy-srv-diff-platforms/serverless/manifest-sources-for-serverless-lambda.md:50]: serverless

**Review by SmritiSatya — APPROVED** [review]

File: docs/continuous-delivery/deploy-srv-diff-platforms/serverless/manifest-sources-for-serverless-lambda.md (hunk 2 above)
Comment (roshnisarangadharan) [reply docs/continuous-delivery/deploy-srv-diff-platforms/serverless/manifest-sources-for-serverless-lambda.md:50]: Keeping it Serverless for consistency

**Pull request merged**